specs/phase0/p2p-interface.md

```diff
@@ -273,9 +276,11 @@ Attestation subnets are used to propagate unaggregated attestations to subsectio
 
 - `committee_index{subnet_id}_beacon_attestation` - These topics are used to propagate unaggregated attestations to the subnet `subnet_id` (typically beacon and persistent committees) to be aggregated before being gossiped to `beacon_aggregate_and_proof`. The following validations MUST pass before forwarding the `attestation` on the subnet.
     - The attestation's committee index (`attestation.data.index`) is for the correct subnet.
+    - The attestation is the first attestation received for the participating validator for the slot, `attestation.data.slot`
```

Comment:
What if there are multiple bits set in the attestation? (Or is this ruled out for now?)

Reply:
got the order messed up. there's a condition below that ensures that just one attester is included.
will fix

Reply:
done